README.rst

```diff
@@ -918,6 +918,25 @@ configuration status of a device changes, this ensures the check reacts
 quickly to events happening in the network and informs the user promptly
 if there's anything that is not working as intended.
 
+Byte Adaptive Measuring in Charts
+---------------------------------
+
+This behavior helps us to show the data and unit of the chart in a more readable way, the units are shown in `B`, `KB`, `MB` and `GB` respectively, and datapoints have same unit to maintain consistency in the chart, summarycircles have different units and are shown in `B`, `KB`, `MB` and `GB` to overview chart summary in a more readable way.
+The chart automatically sets it's unit according to a specific range of values of data, for the quick glance the Y-Axis Legend has same unit as the datapoints which makes it easier to understand the unit of points in the chart.
+
```

Comment:
please break the lines above at a line length similar to the rest of the document, eg:

```
This behavior helps us to show the data and unit of the chart in a more readable way,
the units are shown in `B`, `KB`, `MB` and `GB` respectively, and datapoints have
same unit to maintain consistency in the chart, summarycircles have different units
... etc ...
```